PxWeb/Mappers/CodelistResponseMapper.cs

```diff
@@ -57,6 +57,8 @@ public CodeListResponse Map(Codelist codelist, string language)
                 }
             }
 
+            codeListResponse.Elimination = codelist.Elimination;
+            codeListResponse.EliminationValueCode = codelist.EliminationValue;
 
```

Comment:
@KentMossback you should also map the `AvailableLanguages` to the response object. I am nt getting an languages in the response.

Reply:
Missed that one out, has added that now in the response mapper